### PR TITLE
Session Replaying in Live Preview

### PR DESCRIPTION
**Pull request by wpride**
Let's you re-run your previous session after updating to latest build

This is a remake of https://github.com/dimagi/commcare-hq/pull/14004

@benrudolph @biyeun do you remember what the firefox issue was there?

cross-request: https://github.com/dimagi/formplayer/pull/295

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js [hunk 1]
```diff
@@ -124,6 +124,10 @@ FormplayerFrontend.on('startForm', function (data) {
     FormplayerFrontend.request("clearMenu");
     FormplayerFrontend.Menus.Util.showBreadcrumbs(data.breadcrumbs);
 
+    var urlObject = Util.currentUrlToObject();
+    urlObject.setSessionId(data.session_id);
+    Util.setUrlToObject(urlObject);
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js]: I think this caused a bug last time it was introduced... struggling to remember what it was

**Review by benrudolph — COMMENTED** [review]
My main issue with this is that the history gets erased when you reload your session so it totally negates the back button. this is a fairly confusing user experience (at least it has been for me)

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js (hunk 1 above)
Comment (biyeun) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js]: i think it had something to do with one question per screen. i vaguely remember this too

**Comment by wpride** [comment]
@benrudolph OK think I fixed all the issues except one. When you update and first return to the question, the answer doesn't populate immediately - only once you navigate forward and back does it reappear. Guessing there's some event we just need to publish

**Comment by benrudolph** [comment]
sounds likely, let's offline

**Pull request closed without merging**